staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -682,6 +682,7 @@ func DefaultBuildHandlerChain(apiHandler http.Handler, c *Config) http.Handler {
 	handler = genericapifilters.WithAuditAnnotations(handler, c.AuditBackend, c.AuditPolicyChecker)
 	handler = genericapifilters.WithWarningRecorder(handler)
 	handler = genericapifilters.WithCacheControl(handler)
+	handler = genericapifilters.WithTrace(handler, c.LongRunningFunc)
```

Comment:
don't you have to be before WithRequestInfo? These filters are like shells added to the outside. I.e. requests hit the lower filters first and inner filters last.

Reply:
Yes, you're right, we want request info to be available so I need to move this up. Looks like the trace filter will still bracket all the other filters that might incur latency if I do this, which is the main reason I've tried to make one of the "outermost" filters initially.

Fixing.